cs/filesystem.texy

```diff
@@ -0,0 +1,97 @@
+
+Práce se souborovým systémem - Nette\Utils\FileSystem
+*****************************************************
+
+.[perex]
+[Nette\Utils\FileSystem |api:] je statická třída s užitečnými funkcemi pro práci se souborovým systémem.
+
+
+Všechny příklady předpokládají vytvořený alias:
+
+/--php
+use Nette\Utils\FileSystem;
```

Comment:
Obecne si rikam, jestli tyhle odstavce nejsou zbytecne. Co myslis @bretislavstepan? Vim, ze to je i u jinych utils stranek, jenom tak premyslim nahlas.